embassy-rp/src/pwm.rs

```diff
@@ -106,6 +106,12 @@ impl<'d> Pwm<'d> {
 
         if let Some(pin) = &a {
             pin.gpio().ctrl().write(|w| w.set_funcsel(4));
+            pin.pad_ctrl().modify(|w| {
+                #[cfg(feature = "_rp235x")]
+                w.set_iso(false);
+                w.set_pue(b_pull == Pull::Up);
```

Comment:
using `b_pull` is a bit strange for pin A. Maybe pull should always be false for A, since A is always an output?

Reply:
I can't comment on the `b_pull` issue, however, I have just tested the fix on my board and it does appear to fix the problem. Thank you for such a quick response.

Reply:
Ah, I thought it was weird there was only one pull argument, but wasn't smart enough to go check the DS. Fixed.